prime-router/src/main/kotlin/tokens/TokenAuthentication.kt

```diff
@@ -65,7 +65,7 @@ class TokenAuthentication(val jtiCache: JtiCache) : Logging {
                 logErr(actionHistory, "SenderToken has null JWT ID.  Rejecting.")
                 return false
             }
-            val expiresAt = exp.toInstant().atOffset(ZoneOffset.UTC)
+            val expiresAt = exp.toInstant().atOffset(Environment.rsTimeZone)
             if (expiresAt.isBefore(OffsetDateTime.now())) {
```

Comment:
Should this be explicit about the timezone in the `now` call as well. I think there's a `now(<TimeZone>` call?

Reply:
Not in this case.  isBefore tests the instant, which is timezone/offset agnostic.  You really need to look at the javadoc to figure out is that is the case with a function. For example, compareTo uses more than the instant and timezone/offset which will be a problem.